src/Generation/GapicClientV2Generator.php

```diff
@@ -98,7 +98,11 @@ private function generateImpl(): PhpFile
 
     private function generateClass(): PhpClass
     {
-        return AST::class($this->serviceDetails->gapicClientV2Type)
+        return AST::class(
+                $this->serviceDetails->gapicClientV2Type,
+                null,
+                false,
+                true)
```

Comment:
Let's use PHP8.1 named arguments!
```suggestion
        return AST::class(
                $this->serviceDetails->gapicClientV2Type,
                abstract: true)
```

Reply:
I _think_ the generator supports being run as `7.4` and `8.0` still per: https://github.com/googleapis/gapic-generator-php/blob/main/composer.json#L30

Not sure if that is valid still or not. I'd be happy to bump the PHP version of this to `^8.1` though.

Reply:
The `composer.json` for the V2 branch says that it requires 8.1. I'm not sure if this is accurate or requires more work, though.

Reply:
Oh yes yes. Then we should be good to go with your suggestion! My b

Reply:
Good idea. It completely skipped my mind that we're  using `8.1`.